src/Protocol/Protocol_1_13.cpp

```diff
@@ -395,6 +424,50 @@ void cProtocol_1_13::SendUpdateBlockEntity(cBlockEntity & a_BlockEntity)
 
 
 
+UInt32 cProtocol_1_13::GetProtocolMobID(eMonsterType a_MobType)
+{
+	switch (a_MobType)
+	{
+		case mtInvalidType:           return -1;
```

Comment:
You'll either need to `static_cast` to `UInt32`, or pick a different value for "bad type". Possibly even make it a `static const` in `cProtocol`.